libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.ts

```diff
@@ -146,6 +146,7 @@ export class SkyAutocompleteComponent implements OnDestroy, AfterViewInit {
   /**
    * Specifies the object properties to search.
    * @default ["name"]
+   * @deprecated Use `searchAsync` to search specific properties instead.
```

Comment:
```suggestion
   * @deprecated We recommend against using this property. To search specific properties, use the `searchAsync` event instead.
```
I checked our other deprecated properties, and while we handle them inconsistently, several start with boilerplate text that recommends against using the property. I'd use that boilerplate text here, and I will create a task to clean up the deprecation notes across components to make them more consistent.

Also, I added "the" and "event" around `searchAsync` because I was initially confused when I couldn't find a `searchAsync` property. I figured out that we're referring to the event, but I figured it was worth calling out that it's an even to potentially avoid this initial confusion for consumers.